### PR TITLE
IIDM adders default values

### PR DESCRIPTION
**Pull request by geofjamg**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Technical debt.

**What is the new behavior (if this is a feature change)?**
Following https://github.com/powsybl/powsybl-core/pull/2417, numbers of parameters now have default values and do not need to be provided.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/test/java/com/powsybl/openloadflow/network/ConnectedComponentNetworkFactory.java [hunk 1]
```diff
@@ -140,9 +140,6 @@ public static Network createTwoCcWithATransformerLinkedByASingleLine() {
            .beginStep()
            .setR(0)
```
Comment (annetill) [inline src/test/java/com/powsybl/openloadflow/network/ConnectedComponentNetworkFactory.java:141]: You can remove R here too.

File: src/test/java/com/powsybl/openloadflow/network/ConnectedComponentNetworkFactory.java [hunk 2]
```diff
@@ -189,9 +186,6 @@ public static Network createTwoCcLinkedByATransformer() {
             .beginStep()
             .setR(0)
```
Comment (annetill) [inline src/test/java/com/powsybl/openloadflow/network/ConnectedComponentNetworkFactory.java:187]: You can remove R here too.

File: src/test/java/com/powsybl/openloadflow/network/ConnectedComponentNetworkFactory.java (hunk 1 above)
Comment (geofjamg) [reply src/test/java/com/powsybl/openloadflow/network/ConnectedComponentNetworkFactory.java]: Done

File: src/test/java/com/powsybl/openloadflow/network/ConnectedComponentNetworkFactory.java (hunk 2 above)
Comment (geofjamg) [reply src/test/java/com/powsybl/openloadflow/network/ConnectedComponentNetworkFactory.java]: Done

**Review by annetill — APPROVED** [review]

**Pull request merged**